src/renderer/hooks/useMapServerQuery.js

```diff
@@ -0,0 +1,15 @@
+// @ts-check
+import ky from 'ky/umd'
+import { useQuery } from '@tanstack/react-query'
+
+// local host and global port number
+// @ts-ignore
+const MAP_SERVER_URL = 'http://127.0.0.1:' + window.mapServerPort
+
+export function useMapServerQuery (resourcePath, enabled) {
+  return useQuery({
+    queryKey: [resourcePath],
```

Comment:
not super familiar with react query but based on the [docs](https://react-query-v3.tanstack.com/guides/query-keys) related to query keys, should this be some sort of partitioned array of items based on the path segments of the `resourcePath`? For example if we're trying to get a specific style `/styles/style-id-1`, we'd have something like `queryKey: ['styles', 'style-id-1']`.

Maybe doesn't matter for how we're using it right now, but just curious if that's the more idiomatic approach to the query key

Reply:
yes, you are right, I will update

Reply:
updated via: https://github.com/digidem/mapeo-desktop/pull/708/commits/d165cf34175c8e0bb856056c6f0f2c60ccd1a531